web/app/view/permissions/Devices.js

```diff
@@ -57,6 +57,14 @@ Ext.define('Traccar.view.permissions.Devices', {
             minWidth: Traccar.Style.columnWidthNormal,
             hidden: true,
             filter: 'string'
+        }, {
+            text: Strings.sharedDisabled,
+            dataIndex: 'disabled',
+            flex: 1,
```

Comment:
Can we separate some config into defaults here?

Reply:
Unfortunately not. `default` config also applied to "checkcolumn"  and it will have same width as others. That do not looks good.

Reply:
Right, I remember now.